service/src/test/java/uk/gov/hmcts/reform/fpl/service/docmosis/EPOGenerationServiceTest.java

```diff
@@ -92,6 +174,44 @@ CaseData getCase(OrderStatus orderStatus) {
             .build();
     }
 
+    CaseData getEpoExclusionRequirementCase(OrderStatus orderStatus,
+                                            EPOExclusionRequirementType epoExclusionRequirementType,
```

Comment:
getCaseWithEpoExclusionRequirement might be better

Reply:
done